source/serverless-testing/tests/README.rst

```diff
@@ -0,0 +1,115 @@
+================
+Serverless Tests
+================
+
+.. contents::
+
+----
+
+Introduction
+============
+
+This file describes a subset of existing spec tests as well as some new prose
+tests that drivers MUST use to assert compatibility with Serverless.
+
+Test Configuration
+==================
+
+These tests MUST be run against a live Serverless instance on Atlas. A new
+Serverless instance MUST be created each time the test suite is run, and it MUST
+be deleted once the tests have completed, regardless of their outcome. The
+`serverless directory in the drivers-evergreen-tools repository`_ contains
+scripts for creating and deleting Serverless instances, and the ``config.yml``
+contains an example evergreen configuration that uses them to run the tests.
+
+.. _serverless directory in the drivers-evergreen-tools repository: https://github.com/mongodb-labs/drivers-evergreen-tools/tree/master/.evergreen/serverless
+
+All tests MUST be run with wire protocol compression and authentication
+enabled. Additionally, Serverless requires the usage of the versioned API, so
+the tests MUST be run with a server API version specified.
+
+Required Variables
```

Comment:
I'm wondering what would be the best way to store the values of these for drivers to refer to. The atlas connectivity ones are in a google doc, so maybe we should go that route too.

Reply:
@agolin95: this sounds like another thing that could ultimately benefit from some structure/process

Reply:
Agreed @jmikola thank you for the tag! Taking a reminder to investigate what kind of secret sharing platform options we have available to us.